app/models/subsidy_rule.rb

```diff
@@ -5,10 +5,8 @@
 # Only admins with direct access to the db will be able to update these records
 #
 class SubsidyRule < UuidApplicationRecord
-  # rubocop:disable Rails/InverseOf
-  belongs_to :county, class_name: 'Lookup::County', foreign_key: 'county_id'
-  belongs_to :state, class_name: 'Lookup::State', foreign_key: 'state_id'
-  # rubocop:enable Rails/InverseOf
+  belongs_to :county, class_name: 'Lookup::County'
+  belongs_to :state, class_name: 'Lookup::State'
 
```

Comment:
@weedySeaDragon and @arku can you take a look at this before I merged it?  Per new rubocop version, declaring a default foreign_key is redundant and not necessary:
```
app/models/subsidy_rule.rb:10:51: C: [Corrected] Rails/RedundantForeignKey: Specifying the default value for foreign_key is redundant.
  belongs_to :state, class_name: 'Lookup::State', foreign_key: 'state_id'
```
 and it says disabling inverseof is not necessary
```
app/models/subsidy_rule.rb:8:3: W: [Corrected] Lint/RedundantCopDisableDirective: Unnecessary disabling of Rails/InverseOf.
  # rubocop:disable Rails/InverseOf
```

Just want to make sure this makes sense to you before I merge

Reply:
Yup -- it's completely correct.  (go figure. ;-) )  No `foreign_key` needed and looks like rubocop won't complain about needing `inverseof`.

LGTM!